cobbler/api.py

```diff
@@ -1269,6 +1269,23 @@ def validate_autoinstall_files(self):
 
     # ==========================================================================
 
+    def sync_systems(self, systems, verbose: bool = False):
+        """
+        Take the values currently written to the configuration files in /etc, and /var, and build out the information
+        tree found in /tftpboot. Any operations done in the API that have not been saved with serialize() will NOT be
+        synchronized with this command.
+
+        :param systems: List of specified systems that needs to be synced
+        :param verbose: If the action should be just logged as needed or (if True) as much verbose as possible.
+        """
+        self.log("sync_systems")
+        # load config and data from backend storage
+        self.deserialize()
```

Comment:
@cxfcxf is this really necessary?

Reply:
> @cxfcxf is this really necessary?

not really necessary, this is for the case that if you change directly on backend storage (like edit the json or change database entry) it will still work.
if the full sync is not using deserialize(), then this is not necessary, sorry for the confusion.
the object itself only deserialize() when initialization, for example when you restart cobbler.

my suggestion would be adding a new function for deserialize() in api call, so backend can be synced with main object in memory without restart cobbler with in API call, but thats should not be in this PR.

Reply:
Ok, then @SchoolGuy and I decided to remove it. Your suggestion is indeed a good consideration for future releases.